listener.py

```diff
@@ -1,6 +1,24 @@
+import certifi
 import pymongo
 
+import ortho
+import pair
+from session import Session
+
+# todo: dedup DB connection and read details from env.
 if __name__ == '__main__':
-    cursor = pymongo.MongoClient("mongodb://127.0.0.1:27017").watch()
-    document = next(cursor)
-    print(document)
\ No newline at end of file
+    print("listening")
+    uri = "mongodb+srv://cluster0.t0zld.mongodb.net/myFirstDatabase?authSource=%24external&authMechanism=MONGODB-X509&retryWrites=true&w=majority"
+    client = pymongo.MongoClient(uri,
+                                 tls=True,
+                                 tlsCertificateKeyFile='X509-cert-5204489386261822956.pem', tlsCAFile=certifi.where())
+    while True:
+        with client.watch() as stream:
+            for change in stream:
+                t = change['ns']['coll']
+                if t == 'pairs':
+                    doc = change['fullDocument']
```

Comment:
I would extract doc from lines 20 and 23 into a single assignment before the conditional. I'd also rename `doc` to `document` or `full_document`.